p5js/YOLO/sketch.js

```diff
@@ -14,27 +14,29 @@ let status;
 let objects = [];
 
 function setup() {
-  createCanvas(640, 480);
+  createCanvas(320, 240);
```

Comment:
This does not really matter since YOLO expects a square video of a fix width and height. ml5 creates that video an the rescales the results to the original video you provide

Reply:
Got it, feel free to revert. I think the `size()` method for the `video` element was helping performance but maybe just b/c it required one fewer rescaling time?

Reply:
let's keep this